### PR TITLE
Sync labels to Jira

### PR DESCRIPTION
**Pull request by zacharyr-qb**
Adds the functionality to be able to add labels to newly created Jira bugs. 

A user can add
```
with:
    jira_label: 'red-team,GreenTeam,blueteam'
```
to their GitHub -> Jira workflows to have those labels added to the Jira bugs.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -14,6 +14,11 @@ inputs:
   jira_project:
     description: 'JIRA project key'
     required: true
+  jira_label:
+    description: 'JIRA bug label(s). Value format "red-team,blue-team,GreenTeam", or "red-team"
```
Comment (cmboling) [inline action.yml:18]:
```suggestion
    description: 'JIRA bug label(s). (e.g. value format can be "red-team,blue-team,GreenTeam", or "red-team")
```

**Comment by cmboling** [comment]
I think we've briefly touched on this today, but as mentioned in your comment this is good for new alerts at this point. Existing open Jira issues won't have this new label. Is that something that you'd want? Also if the `jira_label` changes within the workflow, would you want all issues to adhere to the new label that will be synced? I believe this will involve passing the label to `sync.py` as well as `serve.py` Not sure how many other folks are using _serve_ but let's stick to _sync_ since that's seems to be the most commonly used feature.

**Comment by zacharyr-qb** [comment]
> I think we've briefly touched on this today, but as mentioned in your comment this is good for new alerts at this point. Existing open Jira issues won't have this new label. Is that something that you'd want? 

I think it would be most intuitive if we update existing Jira issues with the new labels that have been entered into the workflow.

> Also if the `jira_label` changes within the workflow, would you want all issues to adhere to the new label that will be synced? 

I think this also is in line with the previous question. I think it would definitely be best for us to have all issues (that have been created, or will be created) by the workflow to be aligned with the labels defined in the workflow.

**Comment by cmboling** [comment]
@zacharyr-qb will you smoke test the changes and let me know how it goes?

**Comment by zacharyr-qb** [comment]
Done! All the functionality works great!

**Review by Henko551 — APPROVED** [review]

**Review by Henko551 — CHANGES_REQUESTED** [review]
- [ ] -

**Pull request merged**